packages/console/src/components/ConnectorLogo/index.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ConnectorResponse } from '@logto/schemas';
+import { AppearanceMode } from '@logto/schemas';
+
+import { useTheme } from '@/hooks/use-theme';
+
+type Props = {
+  className?: string;
+  connector: Pick<ConnectorResponse, 'logo' | 'logoDark'>;
```

Comment:
nit: This is no longer a connector object, consider renaming it to `data`. Or simply just split it into two properties: logo and darkLogo

Reply:
renamed